README.md

```diff
@@ -1,6 +1,22 @@
 Prerequisites
 -------------
 
+* enabled virtualization:
+
+  - check if you have `/dev/kvm` directory:
```

Comment:
@lukasz-laguna  It is not a directory, rather the device node. You can also point to https://www.linux-kvm.org/page/FAQ for more information